### ISSUE TITLE
Add Inno Setup installation steps to complete Flutter windows documentation

### ISSUE THREAD

**Issue by SamiaAshraff** [flutter/flutter#115262]
Unable to follow steps on Flutter windows documentation after Inno setup.

@domesticmouse could you please add Inno setup installation steps and also the process of uploading flutter windows build files to the inno setup software?

**Comment by domesticmouse** [comment]
Silly question of the day, what is Inno Setup in this context? I'm guessing https://jrsoftware.org/isinfo.php

/cc @cbracken @loic-sharma

**Comment by loic-sharma** [comment]
/cc @yaakovschectman who will be revisiting the Windows publishing experience in the near future.

> Silly question of the day, what is Inno Setup in this context? I'm guessing [jrsoftware.org/isinfo.php](https://jrsoftware.org/isinfo.php)

Yup that's right. More docs can be found here: https://jrsoftware.org/ishelp/

**Comment by domesticmouse** [comment]
I'm always a tad worried about documenting how to use a third party software solution, as we take on responsibility for keeping up to date with a company that we have no relationship with. Tech debt, of the tech writing kind.

If there are a handful of solutions here, it may be better to tackle with a codelab or a cookbook article format.

/cc @atsansone for your thoughts

**Comment by loic-sharma** [comment]
> If there are a handful of solutions here...

Yup, there are quite a few options here. Off the top of my head there are:

1. Windows Store
2. Squirrel
3. ClickOnce
4. WinSparkle

**Comment by gspencergoog** [comment]
@domesticmouse What about publishing a Medium (or other external site) article on publishing and linking to it? Then at least it's expected to have a "publish date".

When talking about this in desktop triage, @loic-sharma said we should probably have at least one documented path for publishing, and we all agreed. It's not clear what that one solution should be, though.

**Comment by domesticmouse** [comment]
@SamiaAshraff do you want to work together on a medium post for how to publish a Flutter app with Inno Setup?

**Comment by SamiaAshraff** [comment]
Definitely! I’m sure it well help many during the installation.

**Comment by domesticmouse** [comment]
Please ping this thread when you have a first draft. Either share it via medium or as a google doc. Thanks!
